### PR TITLE
fix #50

### LINKED ISSUE #47: Avoid use of deprecated ioutil package in Go implementation
The [ioutil](https://pkg.go.dev/io/ioutil) package in the Go standard libraries has been deprecated since Go 1.16. Alternatives to all the functions provided by that package are present in the [io](https://pkg.go.dev/io) and [os](https://pkg.go.dev/os) packages. The Go implementation should use the [io](https://pkg.go.dev/io) and [os](https://pkg.go.dev/os) packages, and avoid using the [ioutil](https://pkg.go.dev/io/ioutil) package.

### LINKED ISSUE #50: go lint support in CI
go lint support in CI

### PR DESCRIPTION
**Pull request by SamYuan1990**
Fix #47 and Fix #50

for grpc proto and x509 package issue, will fix in other pr.

Signed-off-by: Sam Yuan <yy19902439@126.com>

### CODE REVIEW

**Comment by SamYuan1990** [comment]
@bestbeforetoday , please help review for #47 part. And also new things found in #53 , #52

File: go/e2e/channel_test.go [hunk 1]
```diff
@@ -5,12 +5,11 @@ import (
 	"crypto/x509"
 	"fabric-admin-sdk/pkg/channel"
 	"fmt"
-	"io/ioutil"
 	"os"
 
 	"github.com/hyperledger-twgc/tape/pkg/infra/basic"
-	"github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/ginkgo/v2"
+	ginkgo "github.com/onsi/ginkgo/v2"
```
Comment (bestbeforetoday) [inline go/e2e/channel_test.go:12]: I'm not sure why ginkgo is imported twice. It seems like once should be enough

**Review by bestbeforetoday — APPROVED** [review]
Looks like good removal of `io/ioutil` usage to me. One comment on a duplicate package import but that was there before this PR change.

File: go/e2e/channel_test.go (hunk 1 above)
Comment (SamYuan1990) [reply go/e2e/channel_test.go]: I suppose due to we invoked ginkgo.skip instead of use skip() directly, going to fix it.

**Pull request merged**